src/style.css

```diff
@@ -1 +1,59 @@
 /* Styles go here */
+body {
+  box-sizing: border-box;
```

Comment:
This value is not inherited, body don't have any paddings or margins by default, so this property is unnecessary